docs/agentd/agent_CLI.md

```diff
@@ -0,0 +1,17 @@
+# Building the Agent
+
+1. From within the cmd/aries-agentd directory, run `go build`.
+
+2. The agent executable (aries-agentd) executable should be created in the same directory.
+
+# Agent Parameters
+
+Start the agent with:
+
+`./aries-agentd start`
+
+Parameters can be set by command line arguments or environment variables. Running `./aries-agentd start --help` will print out all the [adjustable parameters](agent_docker_instructions.md#Flags).
```

Comment:
Note: This links to a file that will exist once https://github.com/hyperledger/aries-framework-go/pull/785 is merged in

Reply:
#785 it's merged can you update the link